docs/block-api.md

```diff
@@ -328,7 +328,23 @@ parent: [ 'core/columns' ],
 
 * **Type:** `Object`
 
-Optional block extended support features. The following options are supported, and should be specified as a boolean `true` or `false` value:
+Optional block extended support features. The following options are supported:
+
+- `align` (default `false`): This property adds block controls which allow to change block's alignment. _Important: It doesn't work with dynamic blocks yet._
+
+```js
+// Add the support for block's alignment.
```

Comment:
I think we could be clearer in this documentation what we mean by "all" alignments, both in how this is enabled by a `true` value, but also specifically which values are included in "all" (since the next example is for "pick"ing, but it's not obvious what options are available).

Reply:
Yes, good call. I will clarify as a code comment tomorrow 👍